core/arch/riscv/mm/core_mmu_arch.c

```diff
@@ -36,6 +36,12 @@
 #define debug_print(...) ((void)0)
 #endif
 
+#define FOUR_KB_SHIFT		12
+#define PAGE_SIZE_SHIFT		FOUR_KB_SHIFT
+#define PAGE_SIZE		BIT(PAGE_SIZE_SHIFT)
+#define PAGE_SIZE_MASK		(PAGE_SIZE - 1)
+#define IS_PAGE_ALIGNED(addr)	(((addr) & PAGE_SIZE_MASK) == 0)
```

Comment:
How about using `SMALL_PAGE_MASK` instead?

Reply:
Thanks for suggestion. I removed those unnecessary macros and used the following macro instead.
```c
#define IS_PAGE_ALIGNED(addr) IS_ALIGNED(addr, SMALL_PAGE_SIZE)
```